lib/exogenesis/version.rb

```diff
@@ -1,3 +1,3 @@
 module Exogenesis
-  VERSION = '0.2.0'
+  VERSION = '1.0.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.